assets/js/analytics.js

```diff
@@ -87,10 +87,6 @@ module.exports = function() {
   var rates = null;
   if (cc && cc !== 'USD') {
     rates = loadJSON('/api/exchange-rates/latest.json');
```

Comment:
Lets require the exchange rates file here, so it'll be built into analytics.js statically.